core-web/libs/data-access/src/lib/dot-temp-file-upload/dot-temp-file-upload.service.ts

```diff
@@ -7,17 +7,7 @@ import { catchError, map, pluck, take } from 'rxjs/operators';
 
 import { DotHttpErrorManagerService } from '@dotcms/data-access';
 import { CoreWebService } from '@dotcms/dotcms-js';
-
-export interface DotCMSTempFile {
```

Comment:
Already exist a Interface with the exact same name. @rjvelazco in the other added metadata property.